docs/source/en/tasks/audio_classification.mdx

```diff
@@ -178,15 +229,89 @@ At this point, only three steps remain:
 ...     train_dataset=encoded_minds["train"],
 ...     eval_dataset=encoded_minds["test"],
 ...     tokenizer=feature_extractor,
+...     compute_metrics=compute_metrics,
 ... )
 
 >>> trainer.train()
 ```
+
+Once training is completed, share your model to the Hub with the [`~transformers.Trainer.push_to_hub`] method so everyone can use your model:
+
+```py
+>>> trainer.push_to_hub()
+```
 </pt>
 </frameworkcontent>
 
 <Tip>
 
-For a more in-depth example of how to fine-tune a model for audio classification, take a look at the corresponding [PyTorch notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/audio_classification.ipynb).
+For a more in-depth example of how to finetune a model for audio classification, take a look at the corresponding [PyTorch notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/audio_classification.ipynb).
 
 </Tip>
+
+## Inference
+
+Great, now that you've finetuned a model, you can use it for inference!
+
+Load an audio file you'd like to run inference on. Remember to resample the sampling rate of the audio file to match the sampling rate of the model if you need to!
+
+```py
+>>> from datasets import load_dataset, Audio
+
+>>> dataset = load_dataset("narad/ravdess", split="train")
```

Comment:
Do we want to use this dataset instead of minds for inference?

Reply:
Oops, completely wrong dataset! Thanks for catching this :)